packages/eris-boiler/src/modules/command/map.ts

```diff
@@ -19,6 +19,18 @@ export class CommandMap<C extends Client> extends LoadMap<Command<C>> {
     this.set(loadableObject.name, loadableObject)
   }
 
+  protected _reload (oldLoadableObject: any, loadableObject: any): void {
+    if (!(
+      loadableObject instanceof Command && oldLoadableObject instanceof Command
+    )) {
+      throw TypeError('Invalid Command found during reloading')
```

Comment:
Maybe print what was supplied in the error?

Reply:
not much point on doing so, but I guess out of production you can do a debug log
```js
if (process.env.NODE_ENV !== 'production') {
  logger.warn('cannot load some-command, while reloading')
}
```

Reply:
?? It helps the developer realize what went wrong. Your thing is pointless